src/skill.rs

```diff
@@ -137,3 +137,28 @@ pub const SURVIVAL: SkillId = SkillId(16);
 
 /// Number of core skills.
 pub const RESERVED_SKILLS: u8 = 17;
+
+/// Returns a string representation of an `SkillId`.\
+/// The string is accurate only when using the skills defined by the SRD.
+pub fn srd_skill_string(id: SkillId) -> String {
+    match id.0 {
+        0 => "acrobatics".to_string(),
+        1 => "animal handling".to_string(),
+        2 => "arcana".to_string(),
+        3 => "athletics".to_string(),
+        4 => "deception".to_string(),
+        5 => "history".to_string(),
+        6 => "insight".to_string(),
+        7 => "intimidation".to_string(),
+        8 => "medicine".to_string(),
+        9 => "nature".to_string(),
+        10 => "perception".to_string(),
+        11 => "performance".to_string(),
+        12 => "persuasion".to_string(),
+        13 => "religion".to_string(),
+        14 => "slight of hand".to_string(),
+        15 => "stealth".to_string(),
+        16 => "survival".to_string(),
+        _ => id.0.to_string(),
+    }
```

Comment:
as before, one `to_string` to rule them all

Reply:
I can't find a way to transform a number into a non-temporary `&str` .. whatever I just went with `not supported`.

Reply:
maybe an optional then?

Reply:
or even a panic? lack of proper string representation should be caught early, otherwise there may be `unsupported` in the middle of the game

Reply:
a panic sound good, user shouldn't use this `fn` if he is not using the standard skills